### DISCUSSION TITLE
[PvP] Custom Reductions

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6999] [General]
So,

we had people ask
- why AV has had different tunings compared to other BGs
- why healing / absorbs where reduced so much

we got:
- https://discord.com/channels/788063059926712341/794247581763895316/1226276548844453938
> After a thorough review of the current situation, **all tuning has been removed from Battlegrounds**, while the minimum tuning that was in place before will remain in Arenas and the Boatwrighte's duel-area for now.
We will keep monitoring the situation to ensure a pleasant PvP experience for our players.

The question is: **what are you trying to achieve?**
do you want to offer a 
- tbc like pvp experience
- a wotlk like pvp experience
- a minimum tolerable pvp experience
- a wotlk-prepatch like experience

so far, all the tuning, different tunings in arena, bgs (also different tuning between bgs), seems rather arbitrary from the outside. What do you strive to provide for s2-s4 until we get wotlk?
(wotlk in itself will technically be an issue again because 3.3.5 talents and abilities are designed around s8, not s5 - so the same idea or concept should keep applying there)

**Comment by 55Honey** [comment]
Hello @balleny ,
It is not feasible for us to answer individual questions about roadmaps.
Feel free to catch up with the post related to these subjects on Chromiecraft.com.

Regards

**Reply by balleny** [reply]
apparently not the only one interested in getting some answer to this question
https://discord.com/channels/788063059926712341/801844176416276500/1229479826830659595

**Comment by happymasksalesmen** [comment]
> so far, all the tuning, different tunings in arena, bgs (also different tuning between bgs), seems rather arbitrary from the outside. What do you strive to provide for s2-s4 until we get wotlk? (wotlk in itself will technically be an issue again because 3.3.5 talents and abilities are designed around s8, not s5 - so the same idea or concept should keep applying there)

global nerfs affect classes differently
typically burst classes are hurt more by global damage reduction 
IMO if you want PvP on chromie to be balanced you need to do nerfs/buffs on a class by class basis, and you also need to get real input cause  players giving bad feedback is how global damage nerfs even came into existence 
and its a non issue in wrath
cause all classes have all their utility and itemization is a whole lot better

**Reply by balleny** [reply]
ye, I also raised the topic in the past - but obviously flat changes arent a good solution.
Imagine unnerfed state: everyone can oneshot everyone, some do 1k damage overkill, some do 15k damage overkill.
now you nerf everything flag 50%: half the classes wont be able to "oneshot" any longer, while the other half still oneshots (+x amount overkill).
Without that flat change, everyone had a chance to kill everyone, after not so much.

At the same time, dying in 2 seconds has literally nothing to do with PvP. If the agreed balance is "everything dies in 1 global", you arent talking PvP any more because you practically click 1 button and win or lose - but you aren't "playing" any longer
